web/console/src/casper/index.ts

```diff
@@ -63,34 +63,24 @@ class CasperTransactionService {
     ): Promise<JsonTypes> {
         const contractHashToBytes = await CasperTransactionService.convertContractHashToBytes(contractAddress);
 
-        try {
-            const walletAddressConverted = CLPublicKey.fromHex(this.walletAddress);
-
-            const deployParams = new DeployUtil.DeployParams(walletAddressConverted, 'casper-test', this.gasPrice, this.ttl);
+        const walletAddressConverted = CLPublicKey.fromHex(this.walletAddress);
 
-            const deploy = DeployUtil.makeDeploy(
-                deployParams,
-                DeployUtil.ExecutableDeployItem.newStoredContractByHash(
-                    contractHashToBytes,
-                    entryPoint,
-                    runtimeArgs),
-                DeployUtil.standardPayment(paymentAmount)
-            );
+        const deployParams = new DeployUtil.DeployParams(walletAddressConverted, 'casper-test', this.gasPrice, this.ttl);
```

Comment:
'casper-test' should be moved in config

Reply:
should it removed to backend?

Reply:
only string 'casper-test' 
in constant or enum or config (it depends on what is this string)